UI/lib/ui-header.html

```diff
@@ -1,58 +1,58 @@
-[%
+<!-- prettier-disable -->
+[%#
    # This helper should be included in files which will be served as
    # top-level responses (i.e. documents on their own); this includes
    # UI/login.html, UI/logout.html, UI/main.html and various UI/setup/ pages
 
    # Most LedgerSMB responses are handled by the 'xhr' Dojo module, which
    # *only* needs opening and closing BODY tags to be there (for now).
-   -%]
+-%]
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
-        <title>[% title %]</title>
-        <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
-        [%# HTML Snippet, for import only %]
-        [%
-            # source comment only!
-            #
-            # don't specify a title on the stylesheets: we want them to be
-            # *persistent*
-            # http://www.w3.org/TR/html401/present/styles.html#h-14.3.1
-        %]
-        <link rel="stylesheet" href="js/dojo/resources/dojo.css" type="text/css" />
-        <link rel="stylesheet" href="js/dijit/themes/[% dojo_theme %]/[% dojo_theme %].css" type="text/css" />
-
-        [% IF form.stylesheet %]
-            <link rel="stylesheet" href="css/[% form.stylesheet %]" type="text/css" />
-        [% ELSIF stylesheet %]
-            <link rel="stylesheet" href="css/[% stylesheet %]" type="text/css" />
-        [% END %]
-        [% FOREACH s = include_stylesheet %]
-            <link rel="stylesheet" href="css/[% s %]" type="text/css" />
-        [% END %]
-        [% IF warn_expire %]
-        <script type="text/javascript">
-        window.alert('[% text('Warning:  Your password will expire in [_1]', pw_expires)%]');
-        </script>
-        [% END %]
-        <script type="text/javascript">
-            var dojoConfig = {
-                async: 1,
-                locale: '[% USER.language.lower().replace('_','-') %]',
-                packages: [{"name":"lsmb","location":"../lsmb"}]
-            };
-            var lsmbConfig = {
-                [% IF USER.dateformat %]
-                 "dateformat": '[% USER.dateformat %]'
-                 [% END %]
-            };
-       </script>
-        <script type="text/javascript" src="js/dojo/dojo.js"></script>
-        <script type="text/javascript" src="js/lsmb/main.js"></script>
-        [% FOREACH s = include_script %]
-            <script type="text/javascript" src="[% s %]" ></script>
-        [% END %]
-        <meta name="robots" content="noindex,nofollow" />
+    <title>[% title %]</title>
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
+    [%+# HTML Snippet, for import only %]
+    [%+#
```

Comment:
@ylavoie, are the plus-signs you've introduced here accidental or intended? If intended, what's their purpose?

Reply:
White space chomping options. PRE_CHOMP (`[%x`) and POST_CHOMP (`x%]`) where `x` can be set to:
```
Constant      Value   Tag Modifier
----------------------------------
CHOMP_NONE         0          +
CHOMP_ONE            1          -
CHOMP_COLLAPSE  2          =
CHOMP_GREEDY      3          ~
```

Reply:
That warrants a comment of its own, in the source, I'd say...

Reply:
Done

Reply:
Thanks.